src/app/CommandHandler.cpp

```diff
@@ -147,6 +146,7 @@ CHIP_ERROR CommandHandler::ProcessInvokeRequest(System::PacketBufferHandle && pa
     {
         err = CHIP_NO_ERROR;
     }
+    ReturnErrorOnFailure(invokeRequestMessage.ExitContainer());
     return err;
```

Comment:
I would expect something more like:
```suggestion
    ReturnErrorOnFailure(err);
    ReturnErrorOnFailure(invokeRequestMessage.ExitContainer());
    return CHIP_NO_ERROR;
```
and similar in other places.

Reply:
Done，thanks